angular/src/app/explore/explore.component.html

```diff
@@ -0,0 +1,7 @@
+<app-info></app-info>
+<br>
```

Comment:
you're mixing up document structure with styling here. Would be much nicer to use margins instead of `<br>`

Reply:
+1